benchmarks/graphics_pipeline/build.gradle

```diff
@@ -1,5 +1,6 @@
 ArrayList projectAssets = [
     "benchmarks/shaders/spv",
+    "benchmarks/textures",
```

Comment:
there are some textures under the `Wood` folder, are they used by benchmarks? if so, should we add the subfolder? if not, shall we remove those textures? https://github.com/google/bigwheels/tree/main/assets/benchmarks/textures/Wood

Reply:
I think these are used by the benchmark `texture_transfer_cpu_to_gpu`

Reply:
good to know, thanks

Reply:
I believe when you add a folder. all files inside it and its subfolders are also added, so we shouldn't need to add the `Wood` subfolder.